LEGO1/mxaudiomanager.h

```diff
@@ -0,0 +1,21 @@
+#ifndef MXAUDIOMANAGER_H
+#define MXAUDIOMANAGER_H
+
+#include "mxmediamanager.h"
+
+// VTABLE 0x100dc6e0
+class MxAudioManager : public MxMediaManager
+{
+public:
+  MxAudioManager();
+  virtual ~MxAudioManager() override;
+
+  void LockedReinitialize(MxS8);
```

Comment:
Looks like an internally used function to me, so I'd make it `private` unless there is anything that indicates otherwise